### ISSUE TITLE
Recognizing certain documents in Node.js produces error (`Cannot read properties of undefined`)

### ISSUE THREAD

**Issue by Balearica** [scribeocr/scribe.js#26]
Using the latest version (0.5.0) recognizing certain inputs produces the following error.  We should figure out what is causing this, and add an automated test to prevent it from happening again. 

```
Error: TypeError: Cannot read properties of undefined (reading '0')
    at Worker.<anonymous> (/home/ben/repos/scribejs/node_modules/@scribe.js/tesseract.js/src/createWorker.js:283:15)
    at Worker.emit (node:events:518:28)
    at MessagePort.<anonymous> (node:internal/worker:263:53)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:826:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28)

```

**Comment by Balearica** [comment]
First reported by @bennyzen in #25.

**Comment by Balearica** [comment]
This issue should be resolved in release `0.5.1`.

**Comment by bennyzen** [comment]
Yes, confirmed. It just works now. Awesome. Thank you so much, @Balearica.
